src/t_string.c

```diff
@@ -84,16 +85,23 @@ void setGenericCommand(client *c, int flags, robj *key, robj *val, robj *expire,
         addReply(c, abort_reply ? abort_reply : shared.null[c->resp]);
         return;
     }
+
+    if (flags & OBJ_SET_GET) {
+       getCommand(c);
```

Comment:
```suggestion
       getGenericCommand(c);
```
i don't know how i feel about one command calling another command's entry point.
once that entry point adds some other pre / post operations (like arity check) it can blow up.

since the GET command's implementation is so simple, we can maybe just copy these 4 lines here.
but that can lead to other problems (missing some features that are later added there).

for now, let's just call `getGenericCommand`, same as the existing implementation of `getsetCommand` does.